manifests/windows/agent6.pp

```diff
@@ -7,7 +7,7 @@
   String $agent_version = $datadog_agent::params::agent_version,
   String $service_ensure = 'running',
   String $baseurl = $datadog_agent::params::agent6_default_repo,
-  String $msi_location = 'c:/tmp',
+  String $msi_location = 'C:/Windows/temp',
```

Comment:
This should probably be %TEMP% (i.e. the TEMP environment variable

Reply:
Unfortunately there isn't a straightforward way of using env vars in puppet. Since this runs as admin, I think it is good enough and we can try to get it right on a separate PR. I agree it can fail (eg: Windows installed somewhere else).